packages/sandpack-core/src/resolver/resolver.ts

```diff
@@ -184,12 +200,17 @@ function* resolveNodeModule(
 
       try {
         const pkgFilePath = pathUtils.join(rootDir, pkgSpecifierParts.filepath);
-        const pkgJson = yield* loadPackageJSON(pkgFilePath, opts, rootDir);
+        const rootPkgJson = yield* loadPackageJSON(rootDir, opts);
```

Comment:
Shouldn't this fail if rootPkgJson does not exist? Given that it isn't a node_module in that case

Reply:
This is called in the `resolveNodeModule` so I assumed that this already makes an assumption about being called for something in `node_modules`. Theoretically, it is possible to put manually something into `node_modules` without putting a `package.json` into it though. Should I handle this case? What should happen? Should I simply throw? Is there any logic in place that will catch this error and handle it gracefully?